packages/wrangler/package.json

```diff
@@ -96,6 +96,7 @@
     "node": ">=16.7.0"
   },
   "jest": {
+    "restoreMocks": true,
```

Comment:
Why this flag?

(I've had bad experiences with this particular flag, so I usually suggest manually mocking/unmocking so the behaviour is explicit in the tests. Particularly when you actually _do_ want the mock to persist across tests, which is harder to do when this flag is on.)

Reply:
I can't think of a case where you would want to mock something in a test and still have it in another test. That is what `beforeEach()` is for, right? Persisting anything across tests can lead to state leakage. In particular, what happens if the tests get run in a different order?

In other test runners, like jasmine, this is not only the default, but I think it is not even configurable... 🤔 

If you feel strongly about this then we can remove it. Then we just need to ensure that whenever anyone spies on a function, then there is a `finally` block or `afterEach()` block where that mock is restored.

Reply:
I don't feel strongly about it, let's keep it. I do understand the motivation, and it makes sense. Just wanted to be sure the decision was taken consciously.